.evergreen/run-tests.sh

```diff
@@ -24,11 +25,8 @@ TOPOLOGY=${TOPOLOGY:-server}
 COMPRESSOR=${COMPRESSOR:-}
 SLOW_TESTS_ONLY=${SLOW_TESTS_ONLY:-false}
 
-# JDK6 needs async.type=netty
-if [ "$JDK" == "jdk6" ]; then
-  export ASYNC_TYPE="-Dorg.mongodb.async.type=netty"
-else
-  export ASYNC_TYPE="-Dorg.mongodb.async.type=nio2"
+if [ "$STREAM_TYPE" == "netty" ]; then
```

Comment:
By setting a default above you don't need the if statement: 

`export ASYNC_TYPE="-Dorg.mongodb.test.async.type=$STREAM_TYPE"`

Reply:
Done